MatomoTracker/MatomoUserDefaults.swift

```diff
@@ -99,6 +99,18 @@ internal struct MatomoUserDefaults {
             userDefaults.set(newValue, forKey: MatomoUserDefaults.Key.lastOrder)
         }
     }
+    
+    func reset() {
+        userDefaults.removeObject(forKey: Key.totalNumberOfVisits)
+        userDefaults.removeObject(forKey: Key.currentVisitTimestamp)
+        userDefaults.removeObject(forKey: Key.previousVistsTimestamp)
+        userDefaults.removeObject(forKey: Key.firstVistsTimestamp)
+        userDefaults.removeObject(forKey: Key.clientID)
+        userDefaults.removeObject(forKey: Key.forcedVisitorID)
+        userDefaults.removeObject(forKey: Key.visitorUserID)
+        userDefaults.removeObject(forKey: Key.optOut)
+        userDefaults.removeObject(forKey: Key.lastOrder)
```

Comment:
I could see the risk, once we add a new property that is stored in the user defaults, that we forget to remove it here as well. What do you think about changing the `internal struct Key` to an `enum Key: String, CaseIterable` so we can iterate over all of them here.